.github/workflows/ci-core.yml

```diff
@@ -173,14 +173,25 @@ jobs:
           echo "TIMEOUT=10m" >> $GITHUB_ENV
           echo "COUNT=50" >> $GITHUB_ENV
       - name: Run tests
+        id: run-tests
+        env:
+          OUTPUT_FILE: ./output.txt
+          USE_TEE: false
         run: ./tools/bin/${{ matrix.cmd }} "${{ matrix.split.pkgs }}"
+      - name: Print Filtered Test Results
+        if: failure()
+        uses: smartcontractkit/chainlink-github-actions/go/go-test-results-parsing@2bb3869750e6e1c2d936fe7179784908981548a4 # v2.2.4
+        with:
+          results-file: ./output.txt
+          output-file: ./output-short.txt
       - name: Store logs artifacts
         if: always()
         uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
         with:
           name: ${{ matrix.cmd }}_${{ matrix.split.idx }}_logs
           path: |
             ./output.txt
+            ./output-short.txt
```

Comment:
Do we still need to upload `output.txt` also? Or could we drop it to save the space.

Reply:
I think leaving it for now is a good idea just in case something is missed in the shortened output we can still dig through the full logs. After we have used the shortened output without any need for the full logs for a couple weeks I would feel much better about not keeping it.